_topic_maps/_topic_map.yml

```diff
@@ -3182,13 +3182,13 @@ Topics:
   - Name: Virtual machine networking
     Dir: vm_networking
     Topics:
-    - Name: Using the default pod network with OpenShift Virtualization
+    - Name: Configuring the virtual machine for the default pod network
```

Comment:
Something odd is happening in the rendered preview. It is not the same as this topic name.
![image](https://user-images.githubusercontent.com/31725355/172128853-ca2a0eaf-54ed-4750-8b1f-fb7f4f7ce746.png)

Reply:
Yeah as you said it's Netlify. I'll manually build with the script this time.